backend/slackbot/tests/test.js

```diff
@@ -4,6 +4,114 @@ var assert = require('assert');
 var worker = require('../worker');
 
 describe('worker', function() {
-  console.log(worker);
-  console.log(assert);
+
+  describe('#buildParameters(apps, slackInput)', function() {
+    const mockApps = {
+      'mockApp1*mockStore': {
+
+      },
+      'mockApp2*mockStore': {
+
+      },
+      'mockApp3*mockStore': {
+
+      }
+    };
+    const mockSlackInput = {
+      'startDate': '1/1/2018',
```

Comment:
Good catch, but the reason I wasn't sure how to test days is since it uses the Date object and uses Date-days to get the result, how would I test that since the time it would call Date is different, or should I use a specific endDate and then use the days back from that?

Reply:
Hmmm, that makes it more difficult. You could probably perform the same calculation within the test and make sure the two dates are the same - the exact times on the two dates will be slightly different, but you could maybe `assert` that the difference is within a certain interval or `assert` that the day/month/year is the same.